internal/konnect/protocol.go

```diff
@@ -0,0 +1,96 @@
+package konnect
+
+const (
+	NodeTypeIngressController = "ingress-controller"
+	NodeTypeKongProxy         = "kong-proxy"
+	NodeTypeIngressProxy      = "ingress-proxy"
```

Comment:
Do you mind adding comments for those `const`s what they are going to be used for?

Reply:
added comments here, and removed `ingress-proxy`.